scss/pages/blog.scss

```diff
@@ -43,64 +45,259 @@ body.blog {
-        }
-        
-        .meta {
-          display: flex;
-          align-items: center;
-          justify-content: center;
-        }
-        
-        .user-profile-details {
-          margin-right: 10px;
-        }
+      height: 200px;
+      object-fit: cover;
+    }
+  }
+
+  .topic-link {
+    position: relative;
+    display: block;
+    padding: 0;
+    color: #222;
+    &:hover {
+      text-decoration: none;
+    }
+  }
+  
+  .header {
+    margin: 20px 0 0;
+  }
+  
+  .title {
+    font-size: 1.6rem;
+    margin: 0 0 0.4em;
+    transition: color 0.2s ease-in-out;
+  }
+  
+  .topic-details {
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+  }
+  
+  .excerpt {
+    max-width: 56em;
+    color: #60757e;
+    p {
+      margin-bottom: 1em;
+      display: -webkit-box;
+      overflow-y: hidden;
+      -webkit-line-clamp: 3;
+      -webkit-box-orient: vertical;
+      word-break: break-word;
+    }
+  }
+  
+  .topic-meta {
+    display: flex;
+    align-items: center;
+    padding: 0;
+  }
+  
+  .topic-byline {
+    display: flex;
+    margin: 0 0 0 0;
+    color: #60757e;
+    padding-top: 10px;
+    line-height: 1.2em;
+    font-weight: 400;
+    width: 100%;
+    .author {
+      width: auto;
+    }
+    .bull {
+      display: inline-block;
+      margin: 0 2px;
+      opacity: 0.6;
+      padding: 0 5px 0 5px;
+      line-height: 1.9em;
+    }
+    .date {
+      line-height: 1.9em;
+      font-size: 13px;
+      margin-top: 3px;  
+    }
+    span {
+      margin: 0;
+    }
+  }
+   
+  @media (min-width: 1001px) {
+    .post-card-large {
+      grid-column: 1 / span 3;
+      display: grid;
+      grid-gap: 4vmin;
+      grid-template-columns: 1fr 1fr 1fr;
+      min-height: 280px;
+
+      .title {
+        margin-top: 0;
+        font-size: 3.2rem;
       }
-      
-      .body {
-        @extend .cooked;
-        
-        pre code {
-          overflow: auto;
-          -moz-tab-size: 4;
-          tab-size: 4;
-          
-          &.lang-markdown {
-            white-space: pre-wrap;
-          }
+
+      .excerpt p {
+        margin-bottom: 1.5em;
+        font-size: 1.7rem;
+        line-height: 1.55em;
+        -webkit-line-clamp: 8;
+      }
+
+      .topic-details {
+        justify-content: center;
+      }
+
+      .topic-image {
+        position: relative;
+        grid-column: 1 / span 2;
+        margin-bottom: 0;
+        min-height: 380px;
+        img {
+          position: absolute;
+          width: 100%;
+          height: 100%;
         }
       }
     }
+    .post-card-large:not(.no-image) .header {
+      margin-top: 0;
+    }
+  }
+  .user-top img {
+    margin-right: 6px;
+  }
+  @media (max-width: 500px) {
+    .title {
+      font-size: 1.9rem;
+    }
+    .excerpt {
+      font-size: 1.6rem;
+    }
+  }
+}
+
+.subscribe {
+  position: fixed;
+  z-index: 2;
+  bottom: 20px;
+  display: flex;
+  align-items: center;
+  right: 12px;
+  border-radius: 2px;
```

Comment:
@MeghnaAJ Let's add a border to the subscribe buttons to make them stand out a bit more.